SPECS/vitess/vitess.spec

```diff
@@ -2,8 +2,8 @@
 %bcond_without check
 
 Name:           vitess
-Version:        8.0.0
-Release:        12%{?dist}
+Version:        16.0.2
```

Comment:
This seems to me like a pretty big jump.
We should ensure if this works as expected and does not break any existing users